### DISCUSSION TITLE
Hull size comparison

### DISCUSSION THREAD

**Discussion by Ssiimonn** [brainvisa/brainvisa.github.io#189] [General]
Hello everyone,

Is there a way to determine the hull size used for the BrainVISA calculations?

Kind regards,
Simon

**Comment by denisri** [comment]
Hi,
The hull volume per hemisphere is stored in sulcal graphs, in global attributes (`brain_hull_volume`).
Otherwise there is a process "morphologist/morphometry/brain volumes" which produces a .csv file with brain measurements, but I don't thing it includes this hull directly, if it is what you are looking for. `bothhemi_closed` is what is the most similar to it but it's not calculated the same way.
Denis
